Makefile

```diff
@@ -77,6 +77,11 @@ local-image: clean
 	docker build -f ./build/scheduler/Dockerfile --build-arg ARCH="amd64" --build-arg RELEASE_VERSION="$(RELEASE_VERSION)" -t $(LOCAL_REGISTRY)/$(LOCAL_IMAGE) .
 	docker build -f ./build/controller/Dockerfile --build-arg ARCH="amd64" -t $(LOCAL_REGISTRY)/$(LOCAL_CONTROLLER_IMAGE) .
 
+.PHONY: local-image.arm64v8
+local-image.arm64v8: clean
+	docker build -f ./build/scheduler/Dockerfile --build-arg ARCH="arm64v8" --build-arg RELEASE_VERSION="$(RELEASE_VERSION)" -t $(LOCAL_REGISTRY)/$(LOCAL_IMAGE) .
```

Comment:
For local build, it may make more sense to detect the local GOARCH directly, and then use the single local-image to build it.

Could you incorporate logic of detecting GOARCH to the local-image Makefile entry? `go env GOARCH` should be helpful.